content/22.vaccines.md

```diff
@@ -489,6 +489,31 @@ Additionally, given that severe SARS-CoV-2 may be associated with a dysregulated
 The article [@doi:10.1016/j.cell.2020.04.042] proposes that trained immunity might lead to an earlier and stronger response, which could in turn reduce viremia and the risk of later, detrimental immunopathology.
 While trained immunity is an interesting possible avenue to complement vaccine development efforts through the use of an existing vaccine, additional research is required to assess whether the BCG vaccine is likely to confer trained immunity in the case of SARS-CoV-2.
 
+### Viral evolution and vaccine protection
+
+With these vaccines in place, one concern is how long they will remain effective, especially as the virus continues to evolve.
+Since the start of this pandemic we have already seen multiple variants emerge: B.1.1.7 (emerged in the UK), B.1.351 (emerged in South Africa), P.1 (emerged in Brazil).
+
+Viruses evolve or mutate at different rates.
+Mutation rate is measured as the number of substitutions per nucleotide per cell infected (&mu_{s/n/c)) [@doi:10.1128/JVI.00694-10].
```

Comment:
I'm not sure how this is supposed to render but if it's mu sub s/n/c, I think for Manubot it would be:
```suggestion
Mutation rate is measured as the number of substitutions per nucleotide per cell infected (&mu;~s/n/c~) [@doi:10.1128/JVI.00694-10].
```